packages/theming/theme-tokens/src/getColorTokens.ts

```diff
@@ -0,0 +1,21 @@
+import lightAliasTokens from '@fluentui-react-native/design-tokens-windows/light/tokens-aliases.json';
+import darkAliasTokens from '@fluentui-react-native/design-tokens-windows/dark/tokens-aliases.json';
+import { hcAliasTokens } from './highContrast/tokens-alias';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+import { assertNever } from 'assert-never';
+
+export function getAliasTokens(mode: AppearanceOptions) {
+  if (mode === 'light') {
+    return lightAliasTokens;
+  } else if (mode === 'dark') {
+    return darkAliasTokens;
+  } else if (mode === 'highContrast') {
+    return hcAliasTokens;
```

Comment:
Does Android even have a high contrast mode? I only see the option "High contrast text" on my phone.

Reply:
No, there are no high contrast tokens in android. Android will not hit this code path as the bundle will pick `export function getAliasTokens(mode: AppearanceOptions)` from `getColorTokens.android.ts` added in PR. This api is an in current implementation (packages/theming/default-theme/src/createAliasTokens.ts) - which is moved to "theme-tokens" package, and both android theme package and default theme now use `getAliasTokens` from  here -
`import { getAliasTokens, getShadowTokens } from '@fluentui-react-native/theme-tokens';
`